docs/monitoring.rst

```diff
@@ -0,0 +1,4 @@
+Monitoring
+----------
+
```

Comment:
I thought Camus outputs some useful counters, but my knowledge is not very current. If there are no useful counters, lets get rid of this page.